app/javascript/components/search/controls/KeywordSearch.js

```diff
@@ -13,7 +13,7 @@ import { closeModal } from 'components/search/controls/SearchPanel'
  * optionally takes a 'keywordValue' prop with the initial value for the field
  */
 export default function KeywordSearch({ keywordPrompt }) {
-  const placeholder = keywordPrompt ? keywordPrompt : 'Search title and description text'
+  const placeholder = keywordPrompt ? keywordPrompt : 'Search...'
```

Comment:
Ellipses are good for telling users to wait.  However, as seen in the [Terra style guide](https://www.sketch.com/s/cca3a834-985b-4165-bbae-21f615aebd2d) and almost all UIs, using them in text input or drop-down selection placeholders is unconventional these days.

```suggestion
  const placeholder = keywordPrompt ? keywordPrompt : 'Search'
```